packages/store/addon/-private/system/model/internal-model.ts

```diff
@@ -245,7 +247,12 @@ export default class InternalModel {
   }
 
   isValid() {
-    return this.currentState.isValid;
+    if (false) {
+      return this.currentState.isValid;
+    } else {
+      // assert here
```

Comment:
what is this code branch for? would we assert because we no longer access this state from internalModel?

Reply:
c, because we shouldn't be calling this from inside IM